### DISCUSSION TITLE
change Metdata written back to original File

### DISCUSSION THREAD

**Discussion by THF-Mediapics** [photoprism/photoprism#4636] [Q&A]
Changed metadata is currently not written back to the original file. I change/complete the metadata for my photos. After saving, I download the image and view the metadata in a separate program (e.g. Adobe Brigde / Photomechanics). Unfortunately, Photprism does not write the metadata back to the JPG file. Since I haven't found anything comparable, hence the question, are there currently plans for this function?

**Comment by graciousgrey** [comment]
Writing metadata back to original files is not possible yet but planned for the future:

- https://github.com/photoprism/photoprism/discussions/1092
- https://docs.photoprism.app/user-guide/library/metadata/#export
